app/src/main/res/layout/activity_presentation_irrelevant.xml

```diff
@@ -113,52 +125,59 @@
         android:id="@+id/pres3_imageView1"
         android:layout_width="45dp"
         android:layout_height="45dp"
+        android:layout_marginStart="10dp"
+        android:contentDescription="@string/red_cross"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.071"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintVertical_bias="0.298"
-        app:srcCompat="@drawable/red_cross"
-        android:contentDescription="@string/red_cross" />
+        app:srcCompat="@drawable/ic_gray_cross" />
 
     <ImageView
         android:id="@+id/pres3_imageView2"
         android:layout_width="45dp"
         android:layout_height="45dp"
+        android:layout_marginStart="10dp"
+        android:contentDescription="@string/red_cross"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.071"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintVertical_bias="0.446"
-        app:srcCompat="@drawable/red_cross"
-        android:contentDescription="@string/red_cross" />
+        app:srcCompat="@drawable/ic_gray_cross" />
 
     <ImageView
         android:id="@+id/pres3_imageView3"
         android:layout_width="45dp"
         android:layout_height="45dp"
+        android:layout_marginStart="10dp"
+        android:contentDescription="@string/red_cross"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.071"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintVertical_bias="0.593"
-        app:srcCompat="@drawable/red_cross"
-        android:contentDescription="@string/red_cross" />
+        app:srcCompat="@drawable/ic_gray_cross" />
 
     <CheckBox
         android:id="@+id/pres3_checkBox"
         android:layout_width="138dp"
         android:layout_height="44dp"
         android:contentDescription="@string/tosbox"
         android:text=""
+        android:textColor="@color/light_gray"
+        android:textColorHighlight="@color/persimmon"
+        android:textColorLink="@color/persimmon"
```

Comment:
I'm not sure of the persimmon for that checkbox, it doesn't appear very well on top of the background